gpt4all-chat/src/chatmodel.h

```diff
@@ -446,6 +492,18 @@ class ChatModel : public QAbstractListModel
                     c.consolidatedSources = consolidateSources(sources);
                 }
             }
+            if (version > 9) {
```

Comment:
I've been confused by this in the past; I would prefer if we compared with the first version that supports the data since that's more intuitive.
```suggestion
            if (version >= 10) {
```